testsuite/MDAnalysisTests/core/test_atomselections.py

```diff
@@ -49,168 +42,163 @@
     PSF_NAMD, PDB_NAMD,
     GRO, NUCL, NUCLsel, TPR, XTC,
     TRZ_psf, TRZ,
-    PDB_full,
     PDB_icodes,
 )
 from MDAnalysisTests import make_Universe
 
 import pytest
 
 
-class TestSelectionsCHARMM(TestCase):
-    def setUp(self):
+class TestSelectionsCHARMM(object):
+    @pytest.fixture()
```

Comment:
If a test method modifies a class scoped fixture, the fixture will remain modified across all other test methods.  So if we want to increase the scope of this fixture, we need to check to see if any methods will modify it.  If any test methods do need to modify the fixture, they should use a different version of the fixture which is scoped to only the function.